aptos-move/framework/aptos-framework/sources/aggregator/aggregator.move

```diff
@@ -24,8 +24,7 @@ module aptos_framework::aggregator {
     /// Represents an integer which supports parallel additions and subtractions
     /// across multiple transactions. See the module description for more details.
     struct Aggregator has store {
-        handle: address,
-        key: address,
+        handle: address, key: address,
```

Comment:
It wraps variables, but does nothing with comments?

Reply:
fixed at https://github.com/movebit/movefmt/commit/88e7c98963e3cc88525cf31483f0ba075af4d9f7